frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -54,6 +57,19 @@ class PopupCreateFile extends React.Component {
     }
   }
 
+  handleAllAppChangeLanguage = data => {
+    console.log('%c<PopupCreateFile> Custom event', 'color: #28a745', CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE, data)
+
+    this.setState(prev => ({
```

Comment:
couldn't you user `appContentCustomEventHandlerAllAppChangeLanguage` here ?
It might be a missing refactor